app/validators/answers/number_answer_validator.py

```diff
@@ -1,7 +1,7 @@
 from app.validators.answers.answer_validator import AnswerValidator
 
-MAX_NUMBER = 9999999999
-MIN_NUMBER = -999999999
+MAX_NUMBER = 999_999_999_999_999
+MIN_NUMBER = -99_999_999_999_999
```

Comment:
If this goes ahead before Donna gets an answer - remember to change min to 15 9s. Feel free to resolve this when you change it for merging or if the decision is made to stay 14 👍

Reply:
Cheers ✌️